manifests/params.pp

```diff
@@ -0,0 +1,22 @@
+#
+#
+# == Class: artifactory::params
+#
+# The params class of the artifactory module.
+#
+#
+class artifactory::params (
+    $artifactory_base       = '/var/opt/jfrog/artifactory',
```

Comment:
because the container is static, there's no reason to override anything in the main class ... . actually, it's counter-productive. in this case I just want to have all the params information in a central place.